src/openrct2/world/Climate.cpp

```diff
@@ -241,6 +241,12 @@ bool climate_is_snowing()
     }
 }
 
+bool climate_is_dry()
```

Comment:
All new functions are to be in TitleCase. Do you think this would be more useful to take an input of `WeatherType` and return if it is dry? i.e. the caller would pass in `gClimateCurrent.Weather`? Not sure myself I'll leave for you to decide.

Reply:
You are absolutely correct. I just followed the examples of climate_is_snowing etc. I have now changed the function to `weatherIsDry(WeatherType)` as I feel that name makes a lot more sense as well in this case.

Reply:
Just need to capitalise the w. WeatherIsDry

Reply:
Done in #b29567c